dynamic-plugins.default.yaml

```diff
@@ -524,17 +524,74 @@ plugins:
                   text: Bulk import
   # Global Header
   - package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-header
-    disabled: true
+    disabled: false
     pluginConfig:
+      app:
+        sidebar:
+          search: false
+          settings: false
       dynamicPlugins:
         frontend:
+          default.main-menu-items:
+            menuItems:
+              default.create:
+                title: ''
+          backstage.plugin-notifications:
+            dynamicRoutes:
+              - importName: NotificationsPage
+                path: /notifications
```

Comment:
```suggestion
```

Looks like this configuration isn't allowed here!

Can you please rebase your commit since #2415 is merged and drop this configuration or the commit. See #2416 😅